app/lib/actions/katello/organization/simple_content_access/prepare_content_overrides.rb

```diff
@@ -0,0 +1,36 @@
+module Actions
+  module Katello
+    module Organization
+      module SimpleContentAccess
+        class PrepareContentOverrides < Actions::Base
+          def plan(organization_id)
+            Rails.logger.info "PrepareContentOverrides plan: #{organization_id.inspect}"
+            organization = ::Organization.find(organization_id.to_i)
+            org_name = organization.name
+
+            plan_self(organization_id: organization_id, organization_name: org_name)
```

Comment:
Why the org_name ? Its not used in run
```suggestion
            plan_self(organization_id: organization_id)
```

Reply:
It's used in `humanized_input`